### PR TITLE
Update optional-elements.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/web-page-structure/optional-elements.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/web-page-structure/optional-elements.md?ref=f5c450d5ff8cc04892476af95b4b97f1fc761bf8)

File: web/html/web-page-structure/optional-elements.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ aspects:
   - introduction
 
 links:
-  - '[W3: Optional Elements](https://www.w3.org/TR/html5/syntax.html#optional-tags){documentation}'
+  - '[Optional Elements](https://html.spec.whatwg.org/multipage/syntax.html#optional-tags){documentation}'
```
Comment (kapnobatai137) [inline web/html/web-page-structure/optional-elements.md:16]:
```suggestion
  - '[Optional Tag List](https://html.spec.whatwg.org/multipage/syntax.html#optional-tags){documentation}'
```

File: web/html/web-page-structure/optional-elements.md [hunk 2]
```diff
@@ -46,14 +46,14 @@ Although this isn't a very common practice, Google's HTML styleguide recommends
...
-<!-- Google: Recommended -->
+<!-- Recommended by Google -->
 <!DOCTYPE html>
 <title>Saving money,
 saving bytes</title>
 <p>Qed.
 ```
 
-In order for the document to display properly with all the optional tags omitted, there are various conditions that need to be met for the document to display properly. <!--All of these conditions can be found in [greater detail here](https://www.w3.org/TR/html5/syntax.html#optional-tags)-->
+In order for the document to display properly with all the optional tags omitted, there are various conditions that need to be met. The `<title>` tag cannot be omitted but the `<p>` tag can. All conditions for each element can be found in greater detail in the `Optional elements` link in the `learn more` section.
```
Comment (kapnobatai137) [inline web/html/web-page-structure/optional-elements.md:56]:
```suggestion
> ❗ Not all tags can be omitted. For example, `<title>` will still need an opening and closing tag.

In order for the document to display properly with all the optional tags omitted, there are various conditions that need to be met. Check out the *Learn More* section for a full list of optional tags and conditions.
```

**Comment by kapnobatai137** [comment]
@Stefan-Stojanovic PR is merged so the report can be resolved and you can reply to the comments

**Pull request merged**